public/index.html

```diff
@@ -44,4 +44,5 @@
   <!-- EJS generated server side -->
   <header></header>
   <main id="data"></main>
+  <footer></footer>
```

Comment:
```suggestion
  <footer>
  	<span class="footer-text">
  	  Powered by <b><a target="_blank" href="https://github.com/dashlog/web-ui">Dashlog</a></b>
  	</span>
  </footer>
```
I also wonder if `<strong>` wouldn't be better semantically than `<b>`.

> The `<strong>` element is for content that is of greater importance, while the `<b>` element is used to draw attention to text without indicating that it's more important.

Reply:
Yes, I agree with using `<strong>` tag here. 
I have updated the PR.